README.md

```diff
@@ -26,6 +26,8 @@ Implements some of the basic commands needed to run a simple server.
 |/motd | core.motd | Shows the message of the day.|
 |/op | core.rank | Add a player to the administrator rank.|
 |/plugins | core.plugins | Shows a list of the plugins.|
+|/version | core.version | Shows the server software version.|
+|/about | core.version | Shows the server software version.|
```

Comment:
Please alpha-sort.
I believe this is generated, so sorting things in Info.lua should fix this, too.